android/daemon/src/main/java/org/mozilla/firefox/vpn/daemon/ConnectionHealth.kt

```diff
@@ -242,11 +246,18 @@ class ConnectionHealth(service: VPNService) {
             if (fallbackServerIsReachable) {
                 recordMetrics(ConnectionStability.Unstable)
 
+                if (LocalDateTime.now() < nextPossibleServerSwitch) {
+                    Log.i(TAG, "Want to switch servers, but it has not been enough time since last server switch")

```

Comment:
How often does this connection check run? Just want to be sure we aren't going to spam the logs if there's a long time to wait before the next possible server switch?

Reply:
Every 30 seconds. In a worst case scenario, it would appear in the logs 29 times before a switch. Since we log most other potential routes of the health check (logging *something*) every 30 seconds, this doesn't seem overkill in my opinion.